dts/riscv/gigadevice/gd32vf103.dtsi

```diff
@@ -381,5 +381,29 @@
 			status = "disabled";
 			label = "TIMER_6";
 		};
+
+		dma0: dma@40020000 {
+			compatible = "gd,gd32-dma";
+			reg = <0x40020000 0x400>;
+			interrupt-parent = <&eclic>;
+			interrupts = <30 0 31 0 32 0 33 0 34 0 35 0 36 0>;
+			rcu-periph-clock = <0x500>;
+			dma-channels = <6>;
+			#dma-cells = <1>;
+			status = "disabled";
+			label = "DMA_0";
+		};
+
+		dma1: dma@40020400 {
+			compatible = "gd,gd32-dma";
```

Comment:
I added DMA nodes to gd32e10x, gd32f3x0, gd32f403 and gd32f4xx.
I verified gd32vf103 with longan nano board. (I don't have a these boards.)
I only verified it can compile in these targets.

Reply:
I can help to test gd32f450i_eval, gd32f350r_eval, gd32vf103v_eval, will be done at this weekend.